From 38fd1fb3abf7b83d14be2dcda1969463b5a9121a Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:35:54 -0400
Subject: [block] blk-mq: cleanup blk_mq_init_tags

Message-id: <1402670238-13916-67-git-send-email-snitzer@redhat.com>
Patchwork-id: 83810
O-Subject: [RHEL7.1 PATCH 066/150] blk-mq: bitmap tag: cleanup blk_mq_init_tags
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit 1f236ab22ce3bc5d4f975aa116966c0ea7ec2013
Author: Ming Lei <tom.leiming@gmail.com>
Date:   Sun May 11 01:01:51 2014 +0800

    blk-mq: bitmap tag: cleanup blk_mq_init_tags

    Both nr_cache and nr_tags arn't needed for bitmap tag anymore.

    Signed-off-by: Ming Lei <tom.leiming@gmail.com>
    Signed-off-by: Jens Axboe <axboe@fb.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-mq-tag.c b/block/blk-mq-tag.c
index f196e60..8d526a3 100644
--- a/block/blk-mq-tag.c
+++ b/block/blk-mq-tag.c
@@ -417,7 +417,6 @@ enomem:
 struct blk_mq_tags *blk_mq_init_tags(unsigned int total_tags,
          unsigned int reserved_tags, int node)
 {
- unsigned int nr_tags, nr_cache;
  struct blk_mq_tags *tags;
 
  if (total_tags > BLK_MQ_TAG_MAX) {
@@ -429,9 +428,6 @@ struct blk_mq_tags *blk_mq_init_tags(unsigned int total_tags,
  if (!tags)
   return NULL;
 
- nr_tags = total_tags - reserved_tags;
- nr_cache = nr_tags / num_online_cpus();
-
  tags->nr_tags = total_tags;
  tags->nr_reserved_tags = reserved_tags;
 
-- 
1.7.1